packages/skills/weather_station/skill.yaml

```diff
@@ -0,0 +1,20 @@
+name: weather_station
+authors: Fetch.AI Limited
+version: 0.1.0
+license: Apache 2.0
+url: ""
+behaviours:
+  - behaviour:
+      class_name: MyWeatherBehaviour
+      args:
+        foo: bar
+handler:
```

Comment:
You need to have several handler setup now

Reply:
You might want to declare the `dependencies:` list here.